frontend/src/index.html

```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="icon" type="image/x-icon" href="favicon.ico?v1">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.12/css/all.css" integrity="sha384-G0fIWCsCzJIMAVNQPfjH08cyYaUtMwjJwqiRKxxE/rx96Uroj1BtIQ6MLJuheaO9" crossorigin="anonymous">
```

Comment:
It is really good that you found that font-awesome icons are not working. However, those are probably left-overs from Django to Angular migrations and font-awesome classes should be deleted from the project I guess.

As Weronika wrote, I don't think we should download font-awesome for now. 
Please check 'volontulo-icon' component in our codebase on how to use material icons in the project instead.

Reply:
Actually, we have issue for moving to fontawesome, so maybe this could stay. We'll be changing this soon anyway and removing volontulo-icon component probably.

Reply:
In that case, I am with @magul on importing it as a package to the project and compile it with our scss.

Reply:
@CorneeSean will be done this way, but I think It would be good to do this in another PR and there get rid of other stuff like material icons from all of ours components.